src/App.js

```diff
@@ -1,24 +1,148 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react'
+import PokemonList from './components/PokemonList'
+import Paginator from './components/Paginator'
+import PokemonDetails from './components/PokemonDetails'
+import PokemonCart from './components/PokemonCart'
+import Toolbar from './components/Toolbar';
+import axios from 'axios'
+import {loadStripe} from '@stripe/stripe-js';
+import {Elements} from '@stripe/react-stripe-js';
+import './App.css'
+import SplitForm from './components/SplitForm';
 
 function App() {
+  const endpoint = process.env.REACT_APP_ENDPOINT 
+  const [pokemons, setPokemons] = useState([])
+  const [currentPageUrl, setCurrentPageUrl] = useState(endpoint)
+  const [nextPageUrl, setNextPageUrl] = useState()
+  const [prevPageUrl, setPrevPageUrl] = useState()
+  const [loading, setLoading] = useState(true)
+
+  const [pokemon, setPokemon] = useState('')
+  const [pokemonDetails, setPokemonDetails] = useState([])
+  const [detailsPokemonUrl, setDetailsPokemonUrl] = useState()
+
+  const [cart, setCart] = useState([])
+  const stripePromise = loadStripe(process.env.REACT_APP_TEST_KEY)
+
+  useEffect(() => {
+    async function fetchData() {
+      setLoading(true);
+      let cancel
+      const response = await axios.get(currentPageUrl, {
```

Comment:
I see this code for response duplicated in both effects, you can create a function and just receive the url as param

Reply:
Ready, thanks for the review